src/Symfony/Component/VarDumper/Cloner/VarCloner.php

```diff
@@ -85,9 +85,15 @@ protected function doClone($var)
                 }
             }
             foreach ($vals as $k => $v) {
-                // $v is the original value or a stub object in case of hard references
-                $refs[$k] = $cookie;
-                if ($zvalIsRef = $vals[$k] === $cookie) {
+                if (\PHP_VERSION_ID >= 70400) {
```

Comment:
Can we rely on feature detection instead of version detection (and cache this result)? It seems that `class_exists(ReflectionReference::class)` would be an option.

Reply:
Version detection allows opcache to optimize the generated opcodes.